_posts/2017-08-25-introducing-sandboxfs.md

```diff
@@ -46,51 +52,57 @@ target.
-symlinks involved: sandboxfs exposes real files and directories to the actions,
-so those cannot reach into the original locations.
+symlinks involved. sandboxfs exposes real files and directories to the actions,
+so actions cannot reach into the original locations.
 
 # Isn't FUSE slow though?
 
 Yes. As you may know, [FUSE is slower than a real file
-system](https://www.usenix.org/system/files/conference/fast17/fast17-vangoor.pdf)
-because of the overhead of message-passing between the kernel and userspace,
-and because of all the context switches that are involved in serving a file
-system operation.
+system](https://www.usenix.org/system/files/conference/fast17/fast17-vangoor.pdf).
+This difference is because of the overhead of message-passing between the
+kernel and userspace, and because of all the context switches that are
+involved in serving a file system operation.
 
-Our hypothesis is that, because builds are (generally) not I/O bound, the
+Our hypothesis is that, because builds are generally not I/O bound, the
 increased cost of I/O within the sandbox will pay for itself when compared to
-the cost of actually setting up and tearing down the sandbox for each action
-(no more creating thousands of symlinks nor defining this many mount points).
+the cost of setting up and tearing down the sandbox for each action.
+No more creating thousands of symlinks or defining thousands of mount points.
 
-But for now, that's all that it still is: just a hypothesis. We haven't
-finished stabilizing the sandboxfs code base yet, which means we haven't
-profiled nor tuned it; and the integration points with Bazel are still being
+But for now, that's all that it is: a hypothesis. We haven't
+finished stabilizing the sandboxfs code base, which means we haven't
+profiled nor tuned it. The integration points with Bazel are still being
 defined and implemented, which means it's not yet trivial to test-run Bazel's
 sandboxing with sandboxfs.
 
 # But it's here!
 
-No matter what, there is no reason to keep the code hostage any longer, so we
-are very happy to announce that, as of today, it's now available as an
-open-source project under the Bazel umbrella! See:
+Though still in development, there is no reason to keep the code hostage
+any longer. We are very happy to announce that, as of today, it's now
+available as an open-source project under the Bazel umbrella! See:
 
 > https://github.com/bazelbuild/sandboxfs/
 
-Keep in mind that this is still an experiment and highly subject to change. In
+Keep in mind that this project is still an experiment and highly subject to change. In
 particular, be aware that the command line and the data formats are most
-certainly going to mutate (for simplicity if anything). But, regardless, the
-current code is now sufficient to experiment further and play with.
+certainly going to mutate (for simplicity if anything). But the
+current code is now sufficient to experiment and play with.
 
 Enjoy!
```

Comment:
Is there any specific feedback, info, or help that you want with this new project? You don't need to detail anything here, but if there's something specific you want to hear back about you can add it here.

Would it be helpful if people jump in and help with the development of sandboxfs? If so invite them to do so.

Reply:
Good idea; added something.